src/OpenTelemetry.Instrumentation.AspNetCore/Implementation/HttpInListener.cs

```diff
@@ -294,57 +297,6 @@ public void OnStopActivity(Activity activity, object payload)
             }
         }
 
-        public void OnMvcBeforeAction(Activity activity, object payload)
```

Comment:
Will removing this break < NET6 users? Should we keep it around for `!NET6_0_OR_GREATER`?

Reply:
we could, but that would be inconsistent with metrics part where we collect details during OnStop event.

Reply:
Added it back. Note: This will be inconsistent with metrics for `netstandard2.0` and `netstandard2.1`

Reply:
I'm trying to understand what would be inconsistent with metrics. Is it that the metrics wouldn't have the `http.route` dimension as that because we don't have this information available for metrics?

Reply:
Yes, Having `http.route` in case of traces but not in metrics is the inconsistent part. For `netstandard2.0/2.1`

Reply:
Could we get that from the activity tags and add it as a metric dimension?

Reply:
We could, but that makes the instrumentation more dependent on tracing (Something we would like to avoid)
https://github.com/open-telemetry/opentelemetry-dotnet/issues/2994

Reply:
I think being "more dependent" on it should be okay. As long as we are dependent on`Activity.Current` for duration, we should be fine with using it further for our needs.

In future, when we do get away from depending on `Activity.Current`, we should find a way to populate `http.route` correctly with respect to the new way of doing things.

Reply:
Be careful when depending on Activity for Metric tags. Duration is *okay* as we know the Activity will be created all the time and it will have duration. But that is not necessarily true about Activity Tags - they may/may not be there depending on sampling/filtering.

Reply:
Yeah, and that also means different results for metrics for the case when tracing is enable and when it is not. Right now, users can safely disable tracing without any side effect to metrics

Reply:
We could add the tags to the activity (if not already added by Tracing instrumentation) in the Metrics' listener?

Reply:
We can add some custom tag name `http.route.metrics` and then remove during `Onstop`.  (That way it does not conflict with tracing) (This is extra overhead but it will be only for `netstandard` scenarios). Something similar we do in tracing
https://github.com/open-telemetry/opentelemetry-dotnet/blob/88e8439efa5e833194890d2a569ebd61596a5992/src/OpenTelemetry.Instrumentation.AspNetCore/Implementation/HttpInListener.cs#L271-L275

Reply:
We can do this outside of this PR. Resolving this for now.